src/main/java/org/spongepowered/api/entity/living/player/tab/TabList.java

```diff
@@ -65,6 +65,14 @@
     void setFooter(Text footer);
 
     /**
+     * Sets this list's header and footer.
+     *
+     * @param header The new header
+     * @param footer The new footer
+     */
+    void setHeaderAndFooter(Text header, Text footer);
```

Comment:
The purpose of this method is to prevent calling both individually. In the implementation, the packet is sent with both the header and footer - to send just a header or footer will involve sending an empty text for the opposite, however with this method both will be sent together.